scripts/install

```diff
@@ -226,6 +226,12 @@ ynh_app_setting_set --app=$app --key=registration_shared_secret --value="$regist
 ynh_app_setting_set --app=$app --key=form_secret --value="$form_secret"
 ynh_print_ON
 
+# Create .well-known redirection for access by federation
+if [[ $server_name != $domain ]]
```

Comment:
As we also specify the port in the well-know, I think you should always add this configuration if the `server_name` is a domain managed by yunohost. But in other side you need to check that `$server_name` is managed by yunohost. So I sugest this:

```suggestion
if yunohost --output-as plain domain list | grep -q "^$server_name$"
```